src/components/menu/examples/menu-basic-secondary-text.scss

```diff
@@ -0,0 +1,3 @@
+:host {
+    --menu-surface-width: min(calc(100vw - 4rem), 20rem);
+}
```

Comment:
Would be good to explain how this width works here. Like we did for the previous example  
![image](https://user-images.githubusercontent.com/50618208/151137558-9c14d9be-c1bb-4fd8-b005-188f6312ae7e.png)

Reply:
totally agree. good point! fixup is up